highfidelity/hifi
# RC78: Model collision shape property fix

https://highfidelity.fogbugz.com/f/cases/20735

Pre-merge test plan:
- Enter Interface in desktop mode into any domain with edit permissions
- Open Create and select or create any Model entity
- Verify you can change the model's Collision Shape and Compound Shape (for compound shape setting, a URL to an OBJ file) properties as expected
- Exit and re-enter Create, return to the same model entity from above, verify the Collision Shape and Compound Shape properties are the same as before
- Select or create any Zone entity
- Verify you can change the zone's Shape Type and Compound Shape URL (for use compound shape URL setting, a URL to an OBJ file) properties as expected
- Also verify tooltips appear for Shape Type and Compound Shape URL properties as expected
- Exit and re-enter Create, return to the same zone entity from above, verify the Shape Type and Compound Shape URL properties are the same as before
